test/request_weaver_test.cc

```diff
@@ -46,17 +48,20 @@ class RequestWeaverTest : public ::testing::Test {
     std::vector<std::string> field_names =
         absl::StrSplit(field_path_str, ".", absl::SkipEmpty());
     std::vector<const Field*> field_path;
-    for (const auto& n : field_names) {
+    for (const auto& n: field_names) {
```

Comment:
new style?

Reply:
Reverted. It's auto changed by my clion.